release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1275,6 +1447,55 @@ This update corrects the accepted label specification so that the plug-in import
+This situation occurred because the Prometheus pods had only soft anti-affinity rules regarding node placement.
+Consequently, metrics would not be collected, and rules would not be evaluated until the node came back online.
++
+With this release, the Cluster Monitoring Operator configures hard anti-affinity rules to ensure that the two Prometheus pods are scheduled on different nodes.
+As a result, Prometheus pods are now scheduled on different nodes, and a single node outage no longer creates a gap in monitoring.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1949262[*BZ#1949262*])
+
+* Previously, during {product-title} patch upgrades, the Alertmanager service might become unavailable because either the three Alertmanager pods were located on the same node or the nodes running the Alertmanager pods happened to reboot at the same time. 
+This situation was possible because the Alertmanager pods had soft anti-affinity rules regarding node placement and no pod disruption budget. 
+This release enables hard anti-affinity rules and pod disruption budgets to ensure no downtime during patch upgrades for the Alertmanager and other monitoring components. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955489[*BZ#1955489*])
+
+* Previously, a false positive `NodeFilesystemSpaceFillingUp` alert was triggered when the file system space was occupied by many Docker images. 
+For this release, the threshold to fire the `NodeFilesystemSpaceFillingUp` warning alert is now reduced to 20% space available, instead of 40%, which stops the false positive alert from firing. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987263[*BZ#1987263*])
+
+* Previously, alerts for the Prometheus Operator component did not apply to the Prometheus Operator that runs the `openshift-user-workload-monitoring` namespace when user-defined monitoring is enabled.
+Consequently, no alerts fired when the Prometheus Operator that manages the `openshift-user-workload-monitoring` namespace encountered issues.
++
+With this release, alerts have been modified to monitor both the `openshift-monitoring` and `openshift-user-workload-monitoring` namespaces.
+As a result, cluster administrators receive alert notifications when the Prometheus Operator that manages user-defined monitoring encounters issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001566[*BZ#2001566*])
+
+* Previously, if the number of daemon set pods for the `node-exporter` agent was not equal to the number of nodes in the cluster, the Cluster Monitoring Operator (CMO) would report a condition of `degraded`. This issue would occur when one of the nodes was not in the `ready` condition.
++
+This release now verifies that the number of daemon set pods for the `node-exporter` agent is not less than the number of ready nodes in the cluster. This process ensures that a `node-exporter` pod is running on every active node.
```

Comment:
```suggestion
* Previously, if the number of `daemonset` pods for the `node-exporter` agent was not equal to the number of nodes in the cluster, the Cluster Monitoring Operator (CMO) would report a condition of `degraded`. This issue would occur when one of the nodes was not in the `ready` condition.
+
This release now verifies that the number of `daemonset` pods for the `node-exporter` agent is not less than the number of ready nodes in the cluster. This process ensures that a `node-exporter` pod is running on every active node.
```

`daemonset` is a [kubernetes concept](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). Should it be used as such here?

Reply:
Thanks @jan--f. I'm changing it to DaemonSet, per the k8s usage and our usage elsewhere in the docs

Reply:
to be pedantic, the Kubernetes docs say "DaemonSet".